### ISSUE TITLE
/block returns empty operations

### ISSUE THREAD

**Issue by qiwu7** [Zondax/rosetta-filecoin-proxy#64]
we need to reopen this https://github.com/Zondax/rosetta-filecoin-proxy/issues/61. I am getting empty operations now...

```
{
    "block": {
        "block_identifier": {
            "index": 80608,
            "hash": "0171a0e40220081fb855aeb0746360729437281c873dfd1d4e3a76a1786439373afab33472c9"
        },
        "parent_block_identifier": {
            "index": 80607,
            "hash": "0171a0e4022095efb8b3ca5bd4c2b906d1277dca2d48641a42c2249aefe30ca2b3caa1fd437d"
        },
        "timestamp": 1600724640000,
        "transactions": [
            {
                "transaction_identifier": {
                    "hash": "bafy2bzacebz4sfmw44ubrdmbgcleeeitgdkbws7qy2pom3ddsk6rpapwdawyk"
                },
                "operations": []
            }
        ],
        "metadata": {
            "blockCIDs": [
                "bafy2bzaceavjlozicecq5j7xpse6s5mwjkrle55iax5hnk3qtomatmivehkbe",
                "bafy2bzacectpqmy6w2xigkks7yphpundwzp5bck2hjkucx5b6hxq4ytijm4ju",
                "bafy2bzaceaeortqkz6lepuw6fx7nsce4kkpzelfb6ifxz5wjf4idopo4cmai2"
            ]
        }
    }
}
```

**Comment by qiwu7** [comment]
This is the transaction I send via [filfox](https://filfox.info/en/message/bafy2bzacebxo7ilqhqpinaaesus6plzsm4yfnubhbvpdk2yvosxmwbp4bf3jk) at height 
`81710`. So I queried `81711` via Rosetta block endpoint

```
curl --location --request POST 'http://10.200.128.49:8080/block' \
--header 'Content-Type: application/json' \
--data-raw '{
    "network_identifier": {
        "blockchain": "Filecoin",
        "network": "testnetnet"
    },
    "block_identifier": {
        "index": 81711
    }
}'
```
```
{
    "block": {
        "block_identifier": {
            "index": 81711,
            "hash": "0171a0e402207019e0f2d369bce9110516635e3532c00c22e76791075108ad42a0b213fea466"
        },
        "parent_block_identifier": {
            "index": 81710,
            "hash": "0171a0e402206a9822b22caafea56a9172078a1d45035d2a5802f758f6a46cf0eb2ba0fd255e"
        },
        "timestamp": 1600757730000,
        "transactions": [
            {
                "transaction_identifier": {
                    "hash": "bafy2bzacebxo7ilqhqpinaaesus6plzsm4yfnubhbvpdk2yvosxmwbp4bf3jk"
                },
                "operations": []
            }
        ],
        "metadata": {
            "blockCIDs": [
                "bafy2bzacea3n646mquvityu7jvbgx63zpse3msb5chk5t5s6qztsq2n6k5uwg",
                "bafy2bzaceddxclaxyceseszhnfm4uirz6x3qvfn6der3o56piuy472yevftye",
                "bafy2bzacedmyhe56txrm5ij3kgeivc3ksgomcfg42v4ne77iqrz2h6jcex2qa",
                "bafy2bzacedicyezhremx62bestj7wosufo3y2zwbyyqezimzxoirvtditc4xi",
                "bafy2bzacebsfwm6ox6u7622hwhiur6zpr5w67cfee3qjkugpvrihpff5b5qp4"
            ]
        }
    }
}
```

**Comment by raynaudoe** [comment]
Fixed on #65
